pkg/migrations/sql/20250730124058_update_assessment_table.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE assessments DROP COLUMN source_id;
```

Comment:
Why do we remove it from DB?

Reply:
because of rv tools. either remove or the constraint NOT NULL.

Reply:
Well currently the RVtools to work needs a source to be created. I think it's better option to have RVtools source, and this keep clean and have just connected to `source-id`, ideas?

Reply:
I'll make a design doc for this because IMO the implications are much bigger.